api/src/game_handlers.rs

```diff
@@ -0,0 +1,655 @@
+use crate::client_structs::{
+    ClientCard, CountedCardResponse, CutCardResponse, CutCards, ParsedHand, RandomHandResponse,
+    ScoreResponse,
+};
+use actix_web::{web::Path, HttpResponse, Responder};
+use cribbage_library::cards::Card;
+use cribbage_library::counting::score_counting_cards_played;
+use cribbage_library::cribbage_errors::{CribbageError, CribbageErrorKind};
+use cribbage_library::scoring::{score_hand as scorehand, Score};
+use cribbage_library::select_cards::{get_next_counted_card, select_crib_cards};
+use rand::prelude::{Rng, SliceRandom};
+
+const HOST_NAME: &'static str = "localhost:8088/api"; // important:  no ending '/'
+
+//
+//  cut the cards to see who goes first
+//
+//  sample URLs:
+//              http://localhost:8080/api/cutcards
+//
+//  returns: the two cut cards and the repeat URL.  the client is written to assume a shared notion of the deck
+//           so we just return 2 numbers bewtween 0 and 51
+//
+pub async fn cut_cards() -> impl Responder {
+    let mut rng = rand::thread_rng();
+    let first = rng.gen_range(0..51) as usize;
+    let mut second: usize;
+    loop {
```

Comment:
I know it seems a little less DRY, but I think the intent would be clearer if it looked more like:

```
    let first = rng.gen_range(0..51) as usize;
    let mut second = rng.gen_range(0..51) as usize;
    while first == second {
        second = rng.gen_range(0..51) as usize;
    }
```